documentation/deliberately_vulnerable_flask_app/setup.sh

```diff
@@ -7,11 +7,6 @@
 # Source this file to run it:
 # . ./setup.sh
 
-# This file should ONLY include standard setup steps which would be used by a
-# real Flask project. Do not add anything Pysa/Pyre specific.
-
-python3 -m venv venv
-# shellcheck disable=SC1091
-. venv/bin/activate
-pip install Flask
+pip install -r requirements.txt
 rm ../../.pyre_configuration
+echo '{"source_directories": ["."], "taint_models_path": "../../stubs"}' > ./.pyre_configuration
```

Comment:
I see we didn't detect the XXE vulnerability. This was a problem @PulkitMishra run into and we solved it adding "search_path": "../../stubs"  to the .pyre_configuration make pyre checks the pyi definition there as well (more details in the Discord chat).

Reply:
Got it! Sorry for missing that.